packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -22,10 +22,10 @@
   },
   "categories": ["Languages"],
   "dependencies": {
+    "lwc-language-server": "0.14.2",
+    "vscode-languageclient": "3.5.0",
     "eslint": "4.15.0",
-    "eslint-plugin-lwc": "0.2.0",
-    "lwc-language-server": "0.14.1",
-    "vscode-languageclient": "3.5.0"
+    "eslint-plugin-lwc": "0.2.4"
```

Comment:
@rsalvador - Should I use the latest eslint-plugin-lwc with your changes here?

Reply:
Yes, use version 0.3.2